ci/prow/make_config.go

```diff
@@ -1349,48 +1350,37 @@ func executeTestGroupTemplate(testGroupName string, gcsLogDir string, extras map
 
 // generateDashboard generates the dashboard configuration
 func generateDashboard(repoName string, jobNames []string) {
-	outputConfig(fmt.Sprintf("- name: %s\n  dashboard_tab:", repoName))
+	outputConfig("- name: " + repoName + "\n" + baseIndent + "dashboard_tab:")
+	noExtras := make(map[string]string)
 	for _, jobName := range jobNames {
 		testGroupName := getTestGroupName(repoName, jobName)
-		baseOptions := "sort-by-name="
-		extras := make(map[string]string)
 		switch jobName {
 		case "continuous":
-			dashboardTabName := jobName
-			executeDashboardTabTemplate(dashboardTabName, testGroupName, baseOptions, extras)
-
+			executeDashboardTabTemplate("continuous", testGroupName, testgridTabSortByName, noExtras)
 			// This is a special case for knative/serving, as conformance-tests tab is just a filtered view of the continuous tab.
 			if repoName == "knative-serving" {
-				dashboardTabName := "conformance-tests"
-				baseOptions = "include-filter-by-regex=test/conformance\\\\.&sort-by-name="
-				executeDashboardTabTemplate(dashboardTabName, testGroupName, baseOptions, extras)
+				executeDashboardTabTemplate("conformance-tests", testGroupName, "include-filter-by-regex=test/conformance\\\\.&sort-by-name=", noExtras)
 			}
 		case "dot-release", "auto-release", "performance", "latency", "api-coverage", "playground":
-			dashboardTabName := jobName
-
-			if jobName == "latency" || jobName == "api-coverage" {
-				baseOptions = testgridTabGroupByDir
-			}
-			if jobName == "performance" {
+			extras := make(map[string]string)
+			baseOptions := testgridTabSortByName
+			switch jobName {
+			case "performance":
 				baseOptions = testgridTabGroupByTarget
-			}
-			if jobName == "latency" {
+			case "latency":
+				baseOptions = testgridTabGroupByDir
 				extras["description"] = "95% latency in ms"
-			}
-			if jobName == "api-coverage" {
+			case "api-coverage":
```

Comment:
we recently removed api-coverage as a tab as it was fialing and we did not want to invest looking into it

Reply:
Unrelated, but done. :P